README.md

```diff
@@ -12,17 +12,15 @@ This framework provides for organization of wind and other climatic condition pa
 Furthermore, the framework presents a Digital Exchange Format, consisting of complementary excel and JSON files, embodying the above mentioned reporting requirements and data structure. The excel and JSON files contain identical information. The excel file is useful for human readability, while JSON format is better suited for machine readability. The Digital Exchange Format contains the minimum requirements of data for assessment of the suitability of a wind project development site for any contemplated turbine make/model, and together with the report provides a clear view of project meteorological conditions
 
 ## Usage
-The .xlsx file and Javasript Object Notation (JSON) file are intended to faciliate and streamline data exchange between wind industry stakeholders.  The IEC team has also included a versions that have been populated with example data. 
+The .xlsx file and Javasript Object Notation (JSON) file are intended to faciliate and streamline data exchange between wind industry stakeholders.  The IEC team has also included a versions that have been populated with example data.
```

Comment:
Should read as " The IEC team has also included a version that has been populated with example data."